src/models/thread.ts

```diff
@@ -20,6 +20,11 @@ import { MatrixEvent } from "./event";
 import { EventTimelineSet } from './event-timeline-set';
 import { Room } from './room';
 
+export enum THREAD_EVENTS {
```

Comment:
we normally use Camel Casing for our enum names in the singular pluralisation

Reply:
Done.